app/views/payment_gateways/_braintree_customer_form.html.erb

```diff
@@ -133,4 +124,75 @@
         </div>
       </div>
     </fieldset>
+    <%= hidden_field_tag 'braintree[nonce]', nil, id: 'braintree_nonce' %>
 <% end %>
+
+<script src="https://js.braintreegateway.com/web/3.46.0/js/client.min.js"></script>
+<script src="https://js.braintreegateway.com/web/3.46.0/js/hosted-fields.min.js"></script>
+<script type="text/javascript" charset="utf-8">
```

Comment:
This is redundant with the provider side but that's OK. If anyone wants to refactor we can do later.